src/main/java/com/pushtechnology/diffusion/maven/plugin/AbstractDiffusionMojo.java

```diff
@@ -355,6 +357,9 @@ public void finishConfigurationBeforeStart(ServerConfig config) throws Exception
         // Async pretty useless for testing
         config.getLogging().setAsyncLogging(false);
         config.setMaximumMessageSize(maxMessageSize);
+        mgmt.setEnabled(false);
```

Comment:
Why separate the two added lines? I'd resolve this by in-lining `mgmt`, which also avoids the ugly abbreviation.